lib/sycamore/sycamore/transforms/llm_query.py

```diff
@@ -85,7 +102,7 @@ def _query_text_object(self, object: Union[Document, Element]) -> Union[Document
                     .render(doc=object)
                 )
             else:
-                prompt = self._prompt + "\n" + object["text_representation"]
+                prompt = self._prompt + "\n" + object.text_representation
```

Comment:
Is there a need for this? Would prefer it being standardized

Reply:
text_representation is property, and thought it will be more robust to use if object is missing text representation.